splinepy/io/gismo.py

```diff
@@ -137,44 +144,63 @@ def export(fname, multipatch=None, indent=True):
             [" ".join([str(xx) for xx in x]) for x in interface_array]
         )
 
-    # Start extracting all boundaries
-    # Boundaries are stored as patch-id (global) face-id (local)
-    boundary_spline, boundary_face = np.where(multipatch.interfaces < 0)
-    if boundary_spline.size > 0:
-        boundary_data = ET.SubElement(multipatch_element, "boundary")
-        boundary_data.text = "\n".join(
-            [
-                str(sid + index_offset) + " " + str(bid + 1)
-                for (sid, bid) in zip(boundary_spline, boundary_face)
-            ]
-        )
-    ###
-    # Boundary conditions
-    ###
-    # If there are multiple boundary IDs write them into boundary conditions
-    # and fill with dummy values
-    # Note here, that the boundary conditions are always stored in the local
-    # enumeration system of them multipatch
-    boundary_condition_list = multipatch.boundaries
-    if len(boundary_condition_list) > 1:
-        bcs_data = ET.SubElement(
-            xml_data,
-            "boundaryConditions",
-            multipatch=str(len(multipatch.splines)),
-            id=str(1),
-        )
-        bcs_data.insert(
-            0, ET.Comment(text="Please fill boundary conditions here")
-        )
-        for bc_data_i in boundary_condition_list:
-            bc = ET.SubElement(bcs_data, "bc", type="Dirichlet", unknown="0")
-            bc.text = "\n".join(
+    if labeled_boundaries:
+        # Export geometric boundaries and set a label
+        for boundary_id, face_id_list in enumerate(
+            multipatch.boundaries, start=1
+        ):
+            boundary_data = ET.SubElement(
+                multipatch_element,
+                "boundary",
+                name="BID" + str(boundary_id),
```

Comment:
names are assigned BID# ?
is there way to set an arbitrary name?

Reply:
Not yet, but I can extend thr Multipatch class by a dictionary to specify names.

Reply:
I see, next PR?

Reply:
Should be a two-liner. I can see what I can do in an hour or so...

Reply:
ok, i tried some options, not happy with eiter of those. Next PR please :D